core/object/script_language.h

```diff
@@ -221,20 +221,75 @@ struct ScriptCodeCompletionOption {
 		KIND_FILE_PATH,
 		KIND_PLAIN_TEXT,
 	};
+
+	enum Location {
+		LOCATION_LOCAL = 0,
+		LOCATION_PARENT_MASK = (1 << 8),
+		LOCATION_OTHER_USER_CODE = (1 << 9),
+		LOCATION_OTHER = (1 << 10),
+	};
+
 	Kind kind = KIND_PLAIN_TEXT;
 	String display;
 	String insert_text;
 	Color font_color;
 	RES icon;
 	Variant default_value;
 	Vector<Pair<int, int>> matches;
+	int location;
```

Comment:
This should be assigned a default value since there's a parameter-less constructor.